src/cozmo/util.py

```diff
@@ -722,6 +722,34 @@ def angle_z(self):
         q0,q1,q2,q3 = self.q0_q1_q2_q3
         return Angle(radians=math.atan2(2*(q1*q2+q0*q3), 1-2*(q2**2+q3**2)))
 
+    @property
+    def euler_angles(self):
+        '''class:`Vector3`: The pitch, yaw, roll Euler components of the object's rotation defined as rotations in the x, y, and z axis respectively.'''
+        # convert to matrix
+        matrix = self.to_matrix()
+
+        # normalize the magnitudes of cos(roll)*sin(pitch) (i.e. m12) and cos(roll)*cos(pitch) (ie. m22), to isolate
+        # cos(roll) to be compared against -sin(roll) (m02).  Unfortunately, this omits results with an absolute angle larger than 90 degrees on roll.
```

Comment:
Try to keep lines to about 80 chars max - https://github.com/anki/cozmo-python-sdk/blob/master/CODESTYLE.md